lib/buffer.js

```diff
@@ -637,6 +637,28 @@ Buffer.prototype.includes = function includes(val, byteOffset, encoding) {
   return this.indexOf(val, byteOffset, encoding) !== -1;
 };
 
+// Limit currently not implemented; reserved
+Buffer.prototype.split = function split(sep, limit) {
+  if (!limit)
+    limit = null;
+  if (!sep)
+    return [this];
```

Comment:
very very important that this does return a copy. had a kerfuffle with the `.concat()` method doing the same thing.